deploy/sre-prometheus/100-network-live-migration.PrometheusRule.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-network-live-migration
+    role: alert-rules
+  name: sre-network-live-migration
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-network-live-migration
+    rules:
+    #Expression to check if live migration from SDN to OVN is in-progress for more than
+      - expr:
+            (
+              max(
+                max_over_time(
+                  timestamp(
+                    openshift_network_operator_live_migration_condition{type="NetworkTypeMigrationInProgress"} == 1
+                  )[5d:]
+                )
+              )
+              -
+              min(
+                min_over_time(
+                  timestamp(
+                    openshift_network_operator_live_migration_condition{type="NetworkTypeMigrationInProgress"} == 1
+                  )[5d:]
+                )
+              )
+            )
+          )
+          and on()
+          openshift_network_operator_live_migration_condition{type="NetworkTypeMigrationInProgress"} == 1
+        record: sre:network-live-migration:condition
```

Comment:
Do we know if `sre:...` is valid for getting this metric exported to Telemeter? The card/[CMO regex](https://github.com/openshift/cluster-monitoring-operator/blob/master/Documentation/data-collection.md?plain=1#L38) would seem to indicate that only `cluster:usage...` metrics get this treatment, but its very possible there's other components picking up `sre:...` metrics that I'm not aware of

Regardless, I wouldn't name this recording rule "condition", as I think it's just recording an integer number of seconds of how long the migration is/was in_progress. Perhaps "duration"?

Reply:
Changing to "cluster:usage:network-live-migration:*"